etc/c-icap/c-icap.conf

```diff
@@ -141,7 +141,7 @@ Port 1344
 #       No value
 # Example:
 #       User wwwrun
-User c-icap
+User clamav
```

Comment:
we have to use clamav because other users would not have access to the clamav files ... hm ... if c-icap is a different process it should have read only access to the clamav stuff sa it can never change the virus signatures. But I don't know enough about how the c-icap process works.

Reply:
we use clamav because our container runs with clamav. To start the c-icap server as user `c-icap` we would need sudo permissions in the entrypoint, which i would like to avoid.